### PR TITLE
[TML/BC/OGS] cell finds now finds all cells within the root radius in…

### PR DESCRIPTION
**Pull request by jbathmann**
…stead of the cells right below the tree center

Further, speed is slightly increased as implemented functions from vtk are used.

### CODE REVIEW

**Comment by jvollhueter** [comment]
I tested this successfully on Windows 11, Python 3.9

File: TreeModelLib/BelowgroundCompetition/OGS/helpers/CellInformation.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def getCellId(self, x, y, z):
     #  Suggestions to improveme the implementation are most welcome.
     #  @param x: x-coordinate for tree search
     #  @param y: y-coordinate for tree search
```
Comment (jvollhueter) [inline TreeModelLib/BelowgroundCompetition/OGS/helpers/CellInformation.py:43]: Maybe you could add here the parameter radius for a complete documentation

File: TreeModelLib/BelowgroundCompetition/OGSLargeScale3D/OGSLargeScale3D.py [hunk 2]
```diff
@@ -174,8 +175,9 @@ def addTree(self, tree):
     # respective volume of these cells in tree-own variables.
     # @param x: x-coordinate of tree
     # @param y: y-coordinate of tree
```
Comment (jvollhueter) [inline TreeModelLib/BelowgroundCompetition/OGSLargeScale3D/OGSLargeScale3D.py:177]: Same: Maybe you could add here the parameter radius for a complete documentation

**Review by jvollhueter — COMMENTED** [review]
Apart from including the new radius parameters in the documentation, I haven't noticed anything that could be changed. Great that we can save computing time from now on!

File: TreeModelLib/BelowgroundCompetition/OGS/helpers/CellInformation.py (hunk 1 above)
Comment (jbathmann) [reply TreeModelLib/BelowgroundCompetition/OGS/helpers/CellInformation.py:43]: Added

File: TreeModelLib/BelowgroundCompetition/OGSLargeScale3D/OGSLargeScale3D.py (hunk 2 above)
Comment (jbathmann) [reply TreeModelLib/BelowgroundCompetition/OGSLargeScale3D/OGSLargeScale3D.py:177]: Added

**Comment by jbathmann** [comment]
Thanks for testing and the comments.

**Pull request merged**